android-lib/src/main/java/com/kinvey/android/cache/ClassHash.java

```diff
@@ -695,7 +695,7 @@ static int deleteClassData(String collection, DynamicRealm realm, Class<? extend
             }
             if (fieldInfo.getType().isArray() || Collection.class.isAssignableFrom(fieldInfo.getType())) {
                 Class underlying = getUnderlying(f);
-                if (underlying != null && GenericJson.class.isAssignableFrom(underlying)) {
+                if (underlying != null) {
```

Comment:
Why is the check for GenericJson assignment removed for this fix?

Reply:
Because realm keeps a list of any items like a small table so SDK should have one rule for deleting the list of any types.

Reply:
👍